### DISCUSSION TITLE
Implementation of OnAvailable event handler for CameraView

### DISCUSSION THREAD

**Discussion by Rodionbar** [CommunityToolkit/Maui#2059] [New Feature Discussions]
Dears,

I started a new project with MAUI, and seems that CameraView doesn't have any event handler similar to OnAvailable in Xamarin.

It is pretty useful to put some logic on this event, in my case it's adjusting of camera zoom which currently requires workaround to be implemented. 

Can you pls add this event handler in MAUI?

Thanks in advance!

**Comment by MartinRobins** [comment]
Perhaps this could be implemented as a "scenario" as described in [CameraView frame processing support #1687 here](https://github.com/CommunityToolkit/Maui/discussions/1687)?
